app/code/Magento/Catalog/Test/Unit/Block/Product/ProductList/ToolbarTest.php

```diff
@@ -76,15 +78,20 @@ class ToolbarTest extends TestCase
      */
     protected $pagerBlock;
 
-    protected function setUp(): void
+    /**
+     * @var ToolbarConfig|MockObject
+     */
+    private $toolbarConfig;
+
+    protected function setUp()
```

Comment:
Following change broke the tests. Please fix it like this: 
```suggestion
    protected function setUp(): void
```